src/contracts/interfaces/GPv2ERC1271.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+library ERC1271 {
```

Comment:
uber naming-nit: I would slightly prefer if the library name matched the file name:
```suggestion
library GPv2ERC1271 {
```